### DISCUSSION TITLE
Reproductor de video personalizado con Plyr

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#8] [Announcements]
# Reproductor de video personalizado con Plyr

Personalizar fácilmente reproductor de video y audio de navegadores

https://www.zkreations.com/2018/01/reproductor-html5-responsive-audio-video-plyr.html

**Comment by danieIabel** [comment]
Ya a modo de curiosidad, el corto **Some Thing** de Filmbilder & Friends es uno de mis favoritos, así que en parte use de excusa esta actualización para incluirlo

![bugcat](https://media.tenor.com/WHOwHxdVSQIAAAAC/capoo-capoo-type.gif)

**Comment by danieIabel** [comment]
Para facilitar algunas cosas he modificado el codigo para iniciar Plyr agregando la opción "sin cookies" de youtube, tambien la traduccion de los botones al idioma que prefieras.

**Comment by Fersz4** [comment]
Amigo se podría poner una librería de libass para usar subtitulos en formato ass?

**Reply by danieIabel** [reply]
Que tal amigo, solo es compatible con el formato .srt y .vtt

**Comment by FlexusK** [comment]
Amigo se podría hacer algo para que la calidad del video se elija de manera automática dependiendo la velocidad del internet?

**Reply by danieIabel** [reply]
Lamentablemente Plyr no posee esta característica amigo y tampoco hay una forma fácil de lograrlo, sobre todo por la diferencia entre navegadores.

**Reply by FlexusK** [reply]
Tampoco cuenta con la característica de cambiar de video, me refiero a uno siguiente y uno anterior

**Reply by danieIabel** [reply]
Tampoco existe esa opcion amigo. Las características completas las podrá ver en Github: https://github.com/sampotts/plyr#readme
